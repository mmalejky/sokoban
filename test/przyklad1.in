########
#@-----#
#-w--+-#
#--##--#
#--##--#
#-+--p-#
#------#
########
--------

w2
w8
w6
w4
p8
p8
p8
p4
p4
p4
p6
w2
0
0
0
0
0
0
0
p4
0
0
0
0
w8
w4
p2
p6
.
