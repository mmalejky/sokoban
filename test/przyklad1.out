########
#@-----#
#-w--+-#
#--##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#------#
#-@--+-#
#-w##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#------#
#-w--+-#
#-@##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#------#
#-@w-+-#
#--##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#------#
#-w@-+-#
#--##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#------#
#-w--+-#
#--##--#
#--##p-#
#-+--@-#
#------#
########
--------
########
#------#
#-w--+-#
#--##p-#
#--##@-#
#-+----#
#------#
########
--------
########
#------#
#-w--P-#
#--##@-#
#--##--#
#-+----#
#------#
########
--------
########
#------#
#-w-p*-#
#--##--#
#--##--#
#-+----#
#------#
########
--------
########
#------#
#-wp@+-#
#--##--#
#--##--#
#-+----#
#------#
########
--------
########
#------#
#-wp@+-#
#--##--#
#--##--#
#-+----#
#------#
########
--------
########
#------#
#-wp@+-#
#--##--#
#--##--#
#-+----#
#------#
########
--------
########
#------#
#-@p-+-#
#-w##--#
#--##--#
#-+----#
#------#
########
--------
########
#------#
#-wp@+-#
#--##--#
#--##--#
#-+----#
#------#
########
--------
########
#------#
#-w-p*-#
#--##--#
#--##--#
#-+----#
#------#
########
--------
########
#------#
#-w--P-#
#--##@-#
#--##--#
#-+----#
#------#
########
--------
########
#------#
#-w--+-#
#--##p-#
#--##@-#
#-+----#
#------#
########
--------
########
#------#
#-w--+-#
#--##--#
#--##p-#
#-+--@-#
#------#
########
--------
########
#------#
#-w@-+-#
#--##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#------#
#-@w-+-#
#--##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#------#
#--w-+-#
#--##--#
#--##--#
#-+-p@-#
#------#
########
--------
########
#------#
#-@w-+-#
#--##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#------#
#-w--+-#
#-@##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#------#
#-@--+-#
#-w##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#@-----#
#-w--+-#
#--##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#-w----#
#-@--+-#
#--##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#w@----#
#----+-#
#--##--#
#--##--#
#-+--p-#
#------#
########
--------
########
#w-----#
#----+-#
#--##--#
#--##--#
#-+--@-#
#----p-#
########
--------
########
#w-----#
#----+-#
#--##--#
#--##--#
#-+----#
#----@p#
########
--------
